go.mod

```diff
@@ -71,14 +71,14 @@ require (
 	github.com/sasha-s/go-deadlock v0.3.1
 	github.com/servak/go-fastping v0.0.0-20160802140958-5718d12e20a0
 	github.com/shirou/gopsutil/v3 v3.23.4
-	github.com/sirupsen/logrus v1.9.0
-	github.com/spf13/cast v1.5.0
+	github.com/sirupsen/logrus v1.9.2
```

Comment:
v1.9.1 has one issue, which got reverted in v1.9.2. The diff between v1.9.2 and v1.9.0 seems good ✅ 

https://github.com/sirupsen/logrus/compare/v1.9.0...v1.9.2